src/routes/Project/components/ProjectParams.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+
+import './ProjectParams.scss';
+
+import ProjectConstants from '../constants';
+
+import common from 'common';
+
+export default class ProjectParams extends Component {
+  constructor(props) {
+    super(props);
+    this.handleTimeDivChange = this.handleTimeDivChange.bind(this);
+    this.handleTimeSigChange = this.handleTimeSigChange.bind(this);
+    this.handleTempoChange =   this.handleTempoChange.bind(this);
+  }
+
+  componentDidMount() {
+    ;
+  }
+
+  handleTempoChange(event) {
+    // not supported yet
+    // this.props.updateTempo(event.target.value);
+    ;
+  }
+
+  handleTimeSigChange(event) {
+    // not supported yet
+    // this.props.updateTimeSig(event.target.value);
+    ;
+  }
+
+  handleTimeDivChange(event) {
+    let val = Number(event.target.value);
+    if (2 <= val && val <= 32)
+      this.props.updateTimeDiv(val);
+    event.preventDefault();
+    event.stopPropagation();
+  }
+
+  render() {
+    return (
+      <div className='project-params' id='params-component'>
```

Comment:
ok its fine. you could also just refer to it by component in css, `ProjectParams {... }`